src/app_service/service/dapp_permissions/service.nim

```diff
@@ -29,25 +30,29 @@ method init*(self: Service) =
   try:
     let response = status_go.getDappPermissions()
     for dapp in response.result.getElems().mapIt(it.toDapp()):
-      self.dapps[dapp.name] = dapp
+      self.dapps[dapp.name & dapp.address] = dapp
```

Comment:
This concatenation is being used in different places on this file (line 33, 42, 49, 81). It could be a good idea to extract it to a function

Reply:
I realize the function would be:
```
proc concat(name, address: string): string = 
  return name & address
```
Do u think it adds value agains just name & address ? `&` is already a function ;)